frontend/pages/admin/TeamManagementPage/TeamManagementPage.tsx

```diff
@@ -80,11 +81,16 @@ const TeamManagementPage = (): JSX.Element => {
     (formData: ICreateTeamFormData) => {
       dispatch(teamActions.create(formData))
         .then(() => {
-          dispatch(renderFlash("success", "Team created"));
+          dispatch(
+            renderFlash("success", `Successfully created ${formData.name}.`)
+          );
           dispatch(teamActions.loadAll());
-          // TODO: error handling
         })
-        .catch(() => null);
+        .catch(() => {
+          dispatch(
+            renderFlash("error", "Could not create team. Please try again.")
+          );
+        });
       setShowCreateTeamModal(false);
```

Comment:
hey can we change this to use `toggleCreateModal` function? that has some addition logic to clean up the the `teamEditing` variable and calls `settShowCreateTeamModal itself.

Reply:
Can look at `onDeleteSubmit` for example of calling toggle method.